desc/plotting.py

```diff
@@ -3836,3 +3837,110 @@ def plot_logo(save_path=None, **kwargs):
         fig.savefig(save_path, facecolor=fig.get_facecolor(), edgecolor="none")
 
     return fig, ax
+
+
+def plot_gammac(eq, rhos=None, alphas=None, num_pitch=None, **kwargs):
+    """Plot the energetic proxy metric γ_c for a single flux surface.
+
+    Parameters
+    ----------
+    eq : object
+        Equilibrium object containing magnetic field information
+    rhos : float, optional
+        Flux surface radius. If float, plots single surface.
+        Default: 0.5
+    alphas : array_like, optional
+        Fieldline label values (toroidal angle).
+        Default: np.linspace(0, 2π, 32, endpoint=True)
+    num_pitch : int, optional
+        Number of pitch angle values for bounce integral calculation.
+        Default: 16
+
+    Returns
+    -------
+    fig : matplotlib.figure.Figure
+        The figure object containing the plot
+    ax : matplotlib.axes.Axes
+        The axes object for further customization
+
+    Notes
+    -----
+    An example
+
+    fig, ax = plot_gammac(eq, rhos=0.5)
+    fig.show()
+    """
+    if rhos is None:
+        rhos = np.array([0.5])
+    elif isinstance(rhos, float):
+        rhos = np.array([rhos])
+
+    if alphas is None:
+        alphas = np.linspace(0, 2 * np.pi, 32, endpoint=True)
+
+    if num_pitch is None:
+        num_pitch = 16
+
+    # TODO(#1352)
+    X = kwargs.get("X", 32)
+    Y = kwargs.get("Y", 32)
+    Y_B = kwargs.get("Y_B", 64)
+    num_quad = kwargs.get("num_quad", 24)
+    num_transit = kwargs.get("num_transit", 1)
+
+    figsize = kwargs.get("figsize", (6, 5))
+    cmap = kwargs.get("cmap", "plasma")
```

Comment:
use `kwargs.pop` here to remove them as you go, and then after the last line do
```python
    assert (
        len(kwargs) == 0
    ), f"plot gammac got unexpected keyword argument: {kwargs.keys()}"
```

to catch possible misspelled kwargs etc.

Reply:
Done.